index.html

```diff
@@ -0,0 +1,201 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Lambda School</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Lora" rel="stylesheet">
+  </head>
+  <body>
+  <nav class="Navbar">
+    <ul>
+      <li><a class="NavbarBrand" href="https.://www.lambdaschool.com">&lambda; School</a></li>
```

Comment:
Most people in the industry will scoff at the use of a `capitalized CamelCase class`. For example Navbar, NavbarBrand etc. would be more commonly seen as navbar and navbarBrand.